src/otx/core/data/entity/action_detection.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Module for OTX action data entities."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+from otx.core.data.entity.base import (
+    OTXBatchDataEntity,
+    OTXBatchPredEntity,
+    OTXDataEntity,
+    OTXPredEntity,
+)
+from otx.core.data.entity.utils import register_pytree_node
+from otx.core.types.task import OTXTaskType
+
+if TYPE_CHECKING:
+    from torch import LongTensor
+    from torchvision import tv_tensors
+
+
+@register_pytree_node
+@dataclass
+class ActionDetDataEntity(OTXDataEntity):
+    """Data entity for action classification task.
+
+    Args:
+        bboxes: 2D bounding boxes for actors.
+        labels: One-hot vector of video's action labels.
```

Comment:
What is `frame_path` and `proposals`?

Reply:
frame_path is data media's file path. We need this to get proper meta information of data entity.
Proposals are pre-calculated actor bounding box proposals. Current algorithm saves training time using this kind of pre-calculated actor proposals.

Reply:
You should update them in the docstrings

Reply:
ddb54d7 Updated